pkg/GAPJulia/JuliaInterface/src/JuliaInterface.c

```diff
@@ -330,6 +330,40 @@ static Obj FuncJuliaGetFieldOfObject(Obj self, Obj super_obj, Obj field_name)
     return gap_julia(field_value);
 }
 
+static Obj IsOutputStream;
+
+static Obj FuncStreamViewObj(Obj self, Obj stream, Obj obj)
```

Comment:
Perhaps it would be better to not give the function a CamelCase name which invites people to use it. I'll change it to something like STREAM_VIEWOBJ